src/notifications/celery.py

```diff
@@ -0,0 +1,10 @@
+import os
+
+from celery import Celery
+
+os.environ.setdefault("DJANGO_SETTINGS_MODULE", "notifications.conf.dev")
```

Comment:
look at `src/notifications/wsgi.py` to setup the env - same mechanism should be used here

Reply:
fixed